app/services/analytics.rb

```diff
@@ -30,4 +30,45 @@ def ahoy
   def uuid
     user.uuid
   end
+
+  ACCOUNT_CREATED = 'Account Created'.freeze
+  AUTHENTICATION_ATTEMPT = 'Authentication Attempt'.freeze
+  AUTHENTICATION_ATTEMPT_NONEXISTENT = 'Authentication Attempt with nonexistent user'.freeze
+  AUTHENTICATION_SUCCESFUL = 'Authentication Successful'.freeze
+  EMAIL_CHANGE_REQUESTED = 'User asked to change their email'.freeze
+  EMAIL_CHANGED_AND_CONFIRMED = 'Email changed and confirmed'.freeze
+  EMAIL_CHANGED_TO_EXISTING = 'User attempted to change their email to an existing email'.freeze
+  EMAIL_CONFIRMATION_INVALID_TOKEN = 'Invalid Email Confirmation Token'.freeze
+  EMAIL_CONFIRMATION_TOKEN_EXPIRED = 'Email Confirmation: token expired'.freeze
+  EMAIL_CONFIRMATION_USER_ALREADY_CONFIRMED = 'Email Confirmation: User Already Confirmed'.freeze
+  EMAIL_CONFIRMATION_VALID_TOKEN = 'Email Confirmation: valid token'.freeze
+  GET_REQUEST = 'GET Request'.freeze
```

Comment:
I removed the interpolation on purpose and moved the values to the options hash.

The tracking code takes the extra arguments in the hash and just records them, I checked with @monfresh and it seems like we'll be able to query our analytics by the values in the hash so string interpolation seemed like it would make things harder to query for (and create more event types overall)